tests/utils/random-seed.ts

```diff
@@ -0,0 +1,19 @@
+import crypto from 'crypto';
+import { VERSION } from '../../exports/base';
+
+export const getMagicSeed = () => {
+    const text = `${VERSION}: ${expect.getState().currentTestName ?? ''}`;
```

Comment:
It is not so random. Can use time +` test case name` for random seed?

Reply:
This seed's purpose is to generate "relatively random" numbers. If we use time, it's "too" random: if your tests failed due to a set of random numbers, you're not able to reproduce the failure. Instead, we use engine version and test name: these two quantities can be seen as "relatively stable": you can easily reproduce the failure by knowing the engine version!

Reply:
Ok.